driver-core/src/main/java/com/datastax/driver/core/ProtocolOptions.java

```diff
@@ -135,11 +136,30 @@ public ProtocolOptions(int port) {
      *                        the Cassandra nodes.
      */
     public ProtocolOptions(int port, ProtocolVersion protocolVersion, int maxSchemaAgreementWaitSeconds, SSLOptions sslOptions, AuthProvider authProvider) {
+        this(port, protocolVersion, maxSchemaAgreementWaitSeconds, sslOptions, authProvider, false);
+    }
+
+    /**
+     * Creates a new {@code ProtocolOptions} instance using the provided port
+     * and SSL context.
+     *
+     * @param port            the port to use for the binary protocol.
+     * @param protocolVersion the protocol version to use. This can be {@code null}, in which case the
+     *                        version used will be the biggest version supported by the <em>first</em> node the driver connects to.
+     *                        See {@link Cluster.Builder#withProtocolVersion} for more details.
+     * @param sslOptions      the SSL options to use. Use {@code null} if SSL is not
+     *                        to be used.
+     * @param authProvider    the {@code AuthProvider} to use for authentication against
+     *                        the Cassandra nodes.
+     * @param useBeta         whether or not this client is allowed to connect to servers with protocol versions that are in beta.
+     */
+    public ProtocolOptions(int port, ProtocolVersion protocolVersion, int maxSchemaAgreementWaitSeconds, SSLOptions sslOptions, AuthProvider authProvider, boolean useBeta) {
```

Comment:
Those constructors are really annoying because each new setting requires a new constructor. I wonder if we shouldn't introduce a builder pattern, turn this constructor private and mark the other as `@Deprecated`. This way the private constructor is not part of the API and can be modified any time without risking to break backwards compatibility.